.github/workflows/build.yml

```diff
@@ -25,11 +25,11 @@ jobs:
     - name: decompress ChibiOS ext resources
       run: |
         cd $CH_PATH/ext
-        for i in *.7z; do 7z x -y $i; done
+        for i in *.7z; do 7z x -y "$i"; done
```

Comment:
This is to fix a problem with a whitespace existing in one of the files that gets extracted (pico-sdk).

Reply:
LGTM